recipe/meta.yaml

```diff
@@ -29,10 +27,18 @@ outputs:
         - pkg-config
         - make  # [unix]
       host:
-        - coin-or-cgl
-        - coin-or-clp
-        - coin-or-osi
-        - coin-or-utils
+        - coin-or-cgl 0.60.5
+        - coin-or-clp 1.17.7
+        - coin-or-osi 0.108.7
+        - coin-or-utils 2.11.6
+        - openblas {{ openblas }}         # [blas_impl == "openblas"]
+      run:
+        - bzip2
+        - zlib
+        - readline
```

Comment:
Other coin-or-* packages place `readline` also in `host`. Do we need it in `host` too?